frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -20,15 +20,37 @@ import { ExportButton } from 'EditZing/Components/ExportButton'
 import { Box } from '@mui/material'
 import { CSV_FILE, DOWNLOAD_ALL, HOME_PATH } from '@core'
 
+// import {
+//   CourseInfo,
+//   CourseInfoResponse,
+//   CourseStudentDataResponse,
+//   Group,
+// } from 'EditZing/Types/CourseInfo'
+// import axios, { AxiosResponse } from 'axios'
+
 export const EditZing = () => {
   // get param that was set from history using location
   const history = useHistory()
   const { courseId } = useParams<{ courseId: string }>()
 
   useEffect(() => {
-    if (!courseId) history.push(HOME_PATH)
+    if (!courseId) history.push(courseId)
   }, [courseId, history])
 
+  // const [courseInfo, setCourseInfo] = useState<CourseInfo>()
+
+  // useEffect(() => {
+  //   axios
+  //     .get(`${API_ROOT}${COURSE_API}/${courseId}`)
+  //     .then((res: AxiosResponse<CourseInfoResponse>) => {
+  //       setCourseInfo(res.data.data)
+  //     })
+  //     .catch((error) => {
+  //       console.error(error)
+  //       // setShowError(true)
+  //     })
+  // }, [courseId])
+
```

Comment:
What's going on here? Why is all of this commented out

Reply:
Sorry, I was testing some code from LSC to replace sections of the current code and forgot to remove the commented code. I just removed them and pushed my changes.